crates/router/src/connector/boku/transformers.rs

```diff
@@ -112,47 +268,96 @@ impl<F, T>
     }
 }
 
+fn get_response_status(status: String) -> enums::AttemptStatus {
+    match status.as_str() {
+        "Success" => enums::AttemptStatus::Charged,
```

Comment:
Can we have separate enum for all `BokuStatus`?

Reply:
quickxml doesn't accept enum as data to a field during serializing and deserialize wont recognize enums as field data, instead it creates a self closing field with it. eg. '<data/>'. Thus, had to convert to string to parse into expected format